internal/controller/operatorconfigmap_controller.go

```diff
@@ -613,6 +623,25 @@ func (c *OperatorConfigMapReconciler) getDeployCSIConfig() (bool, error) {
 	return deployCSI, nil
 }
 
+func (c *OperatorConfigMapReconciler) checkForRemoteClient() (bool,error) {
+	clusterVersion := &configv1.ClusterVersion{}
+	clusterVersion.Name = clusterVersionName
+	if err := c.get(clusterVersion); err != nil {
+		c.log.Error(err, "failed to get the clusterVersion version of the OCP cluster")
+		return false, err
+	}
+	storageClients := &v1alpha1.StorageClientList{}
+	if err := c.list(storageClients); err != nil {
+		return false, fmt.Errorf("failed to list storageclients: %v", err)
+	}
+	for idx := range storageClients.Items {
+		if storageClients.Items[idx].Status.ConsumerID == string(clusterVersion.Spec.ClusterID){
+			return false, nil
+		}
+	}
+	return true, nil
```

Comment:
This is a bad indicator for running locally. Provider should be the source of truth for this kind of info.
In the case of subscriptions we can have 1 of 2 options:
- Annotation/label the subscription on the ODF side to indicate quasi-ownership, and look for that annotation here.
- More general annotation approach where an annotation indicates that the client op should not touch the subscription regardless of the reason (I like this more than the first one)
- Have some information passed from the provider to the client as part of the onboarding process to indicate that this client should not bother with noobaa concerns. 

Out of all of the options, I like option number 2 the most. It will give us a mechinizem to disable channel changes for a specific sub-operator. And is flexible enough to allow us in the future to remove the annotation and regain ownership

Reply:
as decided offline, we are going with the approach of a config in the operator's configmap which indicates to the client-operator not to manage noobaa-operator in the case that odf is deployed and the one deploying client-operator and noobaa.